tests/IMAP/AccountTest.php

```diff
@@ -35,7 +35,7 @@ class AccountTest extends AbstractTest {
 	 */
 	public function testCreateAndDelete($name) {
 		$name = uniqid($name);
-		$this->createMailBox($name);
+		$this->getTestAccount()->createMailBox($name);
```

Comment:
This is the method you just removed :see_no_evil:

Reply:
Thanks @ChristophWurst ! Updated in https://github.com/nextcloud/mail/pull/2056/commits/14e2a49057ff613e227adca0aef3a9f2a14ec94b

Reply:
Actually the reason why I implemented the previous method was because I saw this:

https://github.com/nextcloud/mail/blob/c1ef17b8b631bf14ef5a1bc61f2e330f45907b03/tests/IMAP/AbstractTest.php#L83

and 

https://github.com/nextcloud/mail/blob/c1ef17b8b631bf14ef5a1bc61f2e330f45907b03/tests/IMAP/AbstractTest.php#L89

Have I missed out anything?

Reply:
Yes, `getMailbox` still exists. But you removed `createMailbox` from `Account`, hence you can't use it here. Try `$mailbox = $this->getTestAccount()->getImapConnection()->createMailbox(.....)`

Reply:
ok thanks for explaning @ChristophWurst . I have updated in https://github.com/nextcloud/mail/issues/2031